CONTRIBUTING.md

```diff
@@ -1,186 +1,81 @@
-# Node.js Collaborator Guide
+# Node.js Community Contributing Guide 1.0
 
-**Contents**
-
-* [Issues and Pull Requests](#issues-and-pull-requests)
-* [Accepting Modifications](#accepting-modifications)
- - [Involving the Website Admin Group](#involving-the-website-admin-group)
-
-This document contains information for Collaborators of the Node.js
-website project regarding maintaining the code, documentation and issues.
-
-Collaborators should be familiar with the guidelines for new
-contributors in [CONTRIBUTING.md](./CONTRIBUTING.md) and also
-understand the project governance model as outlined in
-[GOVERNANCE.md](./GOVERNANCE.md).
-
-## Issues and Pull Requests
+## Code of Conduct
 
-Courtesy should always be shown to individuals submitting issues and
-pull requests to the Node.js website project.
+The Code of Conduct explains the *bare minimum* behavior
+expectations the Node Foundation requires of its contributors.
+[Please read it before participating.](https://github.com/nodejs/node/blob/master/CODE_OF_CONDUCT.md)
 
-Collaborators should feel free to take full responsibility for
-managing issues and pull requests they feel qualified to handle, as
-long as this is done while being mindful of these guidelines, the
-opinions of other Collaborators and guidance of the Website Admin Group.
+## Vocabulary
 
-Collaborators may **close** any issue or pull request they believe is
-not relevant for the future of the Node.js project. Where this is
-unclear, the issue should be left open for several days to allow for
-additional discussion. Where this does not yield input from Node.js
-Collaborators or additional evidence that the issue has relevance, the
-issue may be closed. Remember that issues can always be re-opened if
-necessary.
+* A **Contributor** is any individual creating or commenting on an issue or pull request.
+* A **Committer** is a subset of contributors who have been given write access to the repository.
```

Comment:
"Committer" was placed here deliberately when the base policy was created: https://github.com/nodejs/TSC/pull/40#discussion_r51919768

I'm currently somewhat confused about these terms though. I've raised an issue in the TSC-repo, where @Fishrock123 hinted we should try to use the term committer more: https://github.com/nodejs/TSC/issues/89#issuecomment-210475348

@fhemberger As mentioned I'm not really sure which of these to use. If you still feel "collaborator" is the better term for us, I'll happily change it.